src/test/java/org/jetbrains/kotlinx/lincheck/test/verifier/nesting_safe_recoverable_linearizability/set/NRLSet.java

```diff
@@ -0,0 +1,234 @@
+package org.jetbrains.kotlinx.lincheck.test.verifier.nesting_safe_recoverable_linearizability.set;
```

Comment:
`nrl` would be a better package name, and the `set` suffix can be removed as well.